docs/terminology.md

```diff
@@ -40,6 +46,10 @@ Constraints are rules that an AdonisFX solver applies during simulation to ensur
 
 Fat or **AdnFat** is an AdonisFX solver for fat simulation. This solver allows to simulate a mesh surface as if it were a closed volume of fat tissue. The volume is constructed procedurally using two compatible geometries: a base mesh to drive the simulation (i.e. the fascia) and a destination mesh (i.e. the fat) on which to apply the simulation. Thanks to that internal volume structure, the solver is able to produce reallistic dynamics typical of fat tissues.
 
+## Glue
+
+Glue or **AdnGlue** is an AdonisFX solver for glueing muscles together after simulation. This solver allows to glue muscles together giving the simulation of the muscles layer a more compact look. The glueing is achieved by ingesting the muscles into the solver and generating one combined output mesh with Glue Constraints applied. Given a maximum glue distance the glueing can be modulated and controlled to reduce the glueing effect against unwanted muscles. Glue can be useful for improving the simulation of the fascia layer by creating a more compact version of the muscles layer avoiding big gaps.
```

Comment:
In UI overview we used "gluing". Can you please double check, confirm which is the right spelling, and fix all occurrences? Thanks!

Reply:
Fixed!